api/src/main/java/com/epam/pipeline/manager/datastorage/DataStorageManager.java

```diff
@@ -311,6 +312,16 @@ public AbstractDataStorage updatePolicy(DataStorageVO dataStorageVO) {
         return createdStorage;
     }
 
+    private StoragePolicy applySpecialRulesToStoragePolicy(final StoragePolicy policy) {
+        final Integer incompleteUploadCleanupDays = preferenceManager
```

Comment:
It seems to me that `policy` maybe nullable here?

Reply:
right, now it is fixed